### PR TITLE
done

### PR DESCRIPTION
**Pull request by yuna-iwata**
created an endpoint which takes a query parameter for the user and outputs the score and date for a particular user by joining the scores and users table

### CODE REVIEW

File: api.py [hunk 1]
```diff
@@ -13,6 +13,18 @@ def index():
     return 'Dino'
 
 
+@app.route('/personal-leaderboard', methods=['GET'])
+def get_personal_leaderboard():
+    username = request.args.get('user', type=str) 
+    data = select("""SELECT score, date, username FROM scores 
+    JOIN users ON  users.user_id = scores.user_id
```
Comment (louisglan) [inline api.py]: very minor but there are two spaces between ON and users

File: api.py [hunk 2]
```diff
@@ -13,6 +13,18 @@ def index():
     return 'Dino'
 
 
+@app.route('/personal-leaderboard', methods=['GET'])
+def get_personal_leaderboard():
+    username = request.args.get('user', type=str) 
+    data = select("""SELECT score, date, username FROM scores 
+    JOIN users ON  users.user_id = scores.user_id 
+    WHERE username =  %s""",(username,))
```
Comment (louisglan) [inline api.py]: two spaces between = and %s here as well

File: db_interactions.py [hunk 3]
```diff
@@ -19,14 +19,14 @@ def insert(query, params=()):
     return "Error inserting data", 500
 
 def select(query, params=()):
-  try:
+  # try:
```
Comment (louisglan) [inline db_interactions.py]: This try catch might be important for other endpoints

**Review by louisglan — CHANGES_REQUESTED** [review]
Just a few minor changes

File: api.py (hunk 1 above)
Comment (yuna-iwata) [reply api.py]: ok

File: api.py (hunk 2 above)
Comment (yuna-iwata) [reply api.py]: how do you even see these hahaha impressive eye

File: db_interactions.py (hunk 3 above)
Comment (yuna-iwata) [reply db_interactions.py]: oops forgot i commented that out my bad

**Review by louisglan — APPROVED** [review]
Noice

**Pull request merged**